inc/commondbtm.class.php

```diff
@@ -36,64 +36,191 @@
+    */
+   public $auto_message_on_action = true;
+
+   /**
+    * Flag to determine whether or not a link to item form can be automatically generated via
+    * self::getLink() method.
+    *
+    * @var boolean
+    *
+    * @TODO This variable should not be updated and could be converted as a constant (protected if min PHP >= 7.1).
+    */
+   public $no_form_page = false;
+
+   /**
+    * Flag to determine whether or not table name of item can be automatically generated via
+    * self::getTable() method.
+    *
+    * @var boolean
+    *
+    * @TODO This variable should not be updated and could be converted as a constant (protected if min PHP >= 7.1).
+    */
+   static protected $notable = false;
+
+   /**
+    * List of fields that must not be taken into account for dictionnary processing.
+    *
+    * @var string[]
+    *
+    * @TODO This variable should not be updated and could be converted as a constant (protected if min PHP >= 7.1 ? it is used in ocsinventoryng).
+    */
+   public $additional_fields_for_dictionnary = [];
 
-   /// Set true to desactivate link generation because form page do not permit show/edit item
-   public $no_form_page                        = false;
+   /**
+    * List of linked item types on which entities informations should be forwarded on update.
+    *
+    * @var string[]
+    *
+    * @TODO This variable should not be updated and could be converted as a constant (protected if min PHP >= 7.1).
+    */
+   static protected $forward_entity_to = [];
+
+   /**
+    * Foreign key field cache : set dynamically calling getForeignKeyField
+    *
+    * @TODO Remove this variable as it is not used ?
+    */
+   protected $fkfield = "";
 
-   /// Set true to desactivate auto compute table name
-   static protected $notable                   = false;
+   /**
+    * Search option of item. Initialized on first call to `self::getOptions()` and used as cache.
+    *
+    * @var array
+    *
+    * @TODO Should be static if used as singleton.
+    */
+   protected $searchopt = false;
 
-   ///Additional fiedls for dictionnary processing
-   public $additional_fields_for_dictionnary   = [];
+   /**
+    * {@inheritDoc}
+    * @see CommonGLPI::$taborientation
+    */
+   public $taborientation = 'vertical';
 
-   /// Forward entity datas to linked items
-   static protected $forward_entity_to         = [];
-   /// Foreign key field cache : set dynamically calling getForeignKeyField
-   protected $fkfield                          = "";
+   /**
+    * {@inheritDoc}
+    * @see CommonGLPI::$get_item_to_display_tab
+    */
+   public $get_item_to_display_tab = true;
 
-   /// Search options of the item : to avoid multiple load
-   protected $searchopt                        = false;
+   /**
+    * List of linked item types from plugins on which entities informations should be forwarded on update.
+    *
+    * @TODO Remove this variable as it is not used ?
```

Comment:
Use inderectly from CommonDBTM::addForwardEntity() call in Plugin class.

Reply:
Yep. I missed it.